meta.go

```diff
@@ -0,0 +1,14 @@
+package tokentpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "token.giantswarm.io"
```

Comment:
I think `token` is a very general name. What about `kube-adm-token` or something more specific?